### PR TITLE
Updated Webpage for Pacific Trails case study

### PR DESCRIPTION
**Pull request by CodyChell**
I didn't do too much design on the CSS side of things. I was following the text and it stopped at the HTML input. It didn't go too in-depth on what design It wanted. Maybe I missing something but I didn't see much specifying the design for CSS. 

Put my own twist on it anyway and put in all the content for the webpages.

### CODE REVIEW

**Comment by CodyChell** [comment]
I realize my mistake now. Named the file half-asleep this morning not thinking that I need to name the homepage file index.html and the yurts.html. Fixed and updated the file naming schemes. Website still doesn't pull the right file though.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+<title>Pacific Trails Resort</title>
+<meta charset="utf-8">
+<link rel="stylesheet" href="style.css"/>
+</head>
+
+    <body>
```
Comment (mbMosman) [inline index.html]: You should use the Brackets Beautify extension (or similar) to cleanup spacing on your files before submission.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        <header>
+            <h1 id="companyname">Pacific Trails Resort</h1>
```
Comment (mbMosman) [inline index.html]: You should not use the companyname style in the header, just paragraph text.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        <nav><a href="index.html" > Home </a> &nbsp;
+            <a href="yurts.html"> Yurts </a> &nbsp;
+            <a href="activites.html"> Activities </a> &nbsp;
+            
+            
+            </nav>    
+        <main>            
+            &nbsp;
+            &nbsp;
+            <div>Welcome to the homepage of pacific trails resort! You'll be able to find many great things here regarding this resort and what it's about. </div>
```
Comment (mbMosman) [inline index.html]: This should not be on the page.  You __must__ not add your own spin to the case study (please check the info on the website for why).  It must be exactly as specified by the requirements on the course website and textbook.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+            
+            <h2 id="header2">Enjoy Nature in Luxury</h2>
```
Comment (mbMosman) [inline index.html]: The id is not needed here.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+                <li>Nightly fine dining at the Overlook Cafe.</li>
+                <li>Heated outdoor pool and whirlpool.</li>
+                <li>Guided hiking tours of the redwoods.</li>
+            </ul>
+                
+            <div>
+                    Pacific Trails Resort <br>
+                    12010 Pacific Trails Raod <br>
+                    Zephyr, CA 95555 <br>
+                    888-555-5555 <br>
```
Comment (mbMosman) [inline index.html]: You should separate the phone number from the address with an extra blank line.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+                </div>
+                
+            
+            &nbsp;
```
Comment (mbMosman) [inline index.html]: This should not be here.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+    
+    
+    
+    
+    
+        </main>
+         <footer>
+             <p>
+                 Copyright &copy; 2016 Pacific Trails Resort <br>
+                 Contact me at my email: Cody@Chell.com
```
Comment (mbMosman) [inline index.html]: The email should be a mail to link.

File: style.css [hunk 8]
```diff
@@ -0,0 +1,13 @@
+body{
```
Comment (mbMosman) [inline style.css]: Make sure that you are using the information from the web site to understand what is part of each assignment:  https://htc-ccis1301.github.io/spring-2017/morea/cs-pac-trails/ex-cs-pt-part1.html 

There is information for the CSS in the assignment.  It is at the end of Ch 4 in your textbook.

File: yurts.html [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+        </header>
+
+        <nav>   
+                 <a href="index.html" > Home </a> &nbsp;
+                 <a href="yurts.html"> Yurts </a> &nbsp;
+                 <a href="activities.html"> Activities </a> &nbsp;
+            </nav>
+        <main>
+        <h2>What is a Yurt?</h2>
+        <dl>
```
Comment (mbMosman) [inline yurts.html]: Note the following from the website instructions: 
>Yurts Page (Major Exception from Text)
>For the Yurts page, the book asks you to use the `<dl>`, `<dd>`, and `<dt>` tags. These don’t make for a very nice display and aren’t very semantically correct. Instead use `<h3>` and `<p>` tags. The questions go in the h3 elements, and the text below should go inside a paragraph element. I will look for this as I am grading, so don’t forget to make this change.

File: yurts.html [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+        
+        
+        
+        
+        </main>
+        
+        
+<footer>
+<p>
+    <small><i>Copyright &copy; 2016 Pacific Trails Resort</i></small> <br>
```
Comment (mbMosman) [inline yurts.html]: You should not use the `<small>` and `<i>` tags in the footer as they are non-semantic HTML.

**Pull request closed without merging**